scopesim/effects/data_container.py

```diff
@@ -102,14 +114,13 @@ def __init__(self, filename=None, table=None, array_dict=None, cmds=None,
         })
         self.meta.update(kwargs)
 
-        self.headers = []
+        self._headers = []  # TODO: What is this attribute used for??
```

Comment:
I'm a bit worried that `.headers` is used somewhere. But it doesn't seem like it? Couldn't find anything in the irdb or ScopeSim_Templates, so it should be fine to rename (and maybe later remove) it.